dotnet/src/Extensions/Experimental/Orchestration.Flow/Experimental.Orchestration.Flow.csproj

```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- THIS PROPERTY GROUP MUST COME FIRST -->
+    <AssemblyName>Microsoft.SemanticKernel.Experimental.Orchestration.Flow</AssemblyName>
+    <RootNamespace>Microsoft.SemanticKernel.Experimental.Orchestration</RootNamespace>
```

Comment:
We prefer if `AssemblyName` and `RootNamespace` match

Reply:
We were following the pattern with Planners.Core.csproj -- will this be changing?

Reply:
Things that ended in .Core were special. Let's leave as is for now.